decentraland/unity-renderer
# fix: fade in/out messages for preview window

## What does this PR change?

Changes the way messages are fade in/out during preview mode. Adds a timer to make the messages disappear when they should when hover is lost.

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=fix/fadein-fadeout-chat-message
2. Open the chat window
3. Click on the chat window area. This should open the chat window.
3. Remove focus from chat window. After 3 secs, it should go to preview window. After 5 secs, all messages should fade out.
4. Receive a public message. If chat window is off, messages should fade in. After 5 secs, all messages should disappear. 
5. Chat window should not dissapear if hover is on. Also, chat window should not dissaper if input field is selected.
6. Preview window should not dissapear if hover is on.

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
